substrate/frame/referenda/src/lib.rs

```diff
@@ -762,8 +789,17 @@ impl<T: Config<I>, I: 'static> Polling<T::Tally> for Pallet<T, I> {
 	) -> Result<R, DispatchError> {
 		match ReferendumInfoFor::<T, I>::get(index) {
 			Some(ReferendumInfo::Ongoing(mut status)) => {
-				let result = f(PollStatus::Ongoing(&mut status.tally, status.track))?;
 				let now = frame_system::Pallet::<T>::block_number();
+				let is_finalizing = status
```

Comment:
Consider making this a helper method possibly with the `#[inline]` hint.

Reply:
Awesome. Let me check how do I do that.